README.md

```diff
@@ -90,7 +95,21 @@ Options:
         Set log level to Trace
 ```
 
-`nancy` currently works for projects that use `dep` or `go mod` for dependencies.
+#### What is the best usage of Nancy?
+
+There are four different ways to use Nancy:
+
+Preferred (will be around in Nancy 1.0.0)
+- `go list -json -m all | nancy`
+- `nancy /path/to/Gopkg.lock`
+
+Unpreferred (will be deprecated in Nancy 1.0.0)
+- `go list -m all | nancy`
+- `nancy /path/to/go.sum`
+
+The reasons for this are myriad, and are:
+- `go.sum` files are not lockfiles, they can contain many entries that your project no longer uses, and are thus, not totally reliable for a source of "what is my project using right now". We have yet to remove this functionality, mostly so we can gracefully remove it when Nancy 1.0.0 arrives (soon!)
```

Comment:
Not a big deal if we dont want to pull the trigger on it but i thought we were removing go.sum support in 1.0.0. 

I have the PR ready to roll too 
https://github.com/sonatype-nexus-community/nancy/pull/155

@DarthHater any thoughts or feelings?? I could go either way if we think there is a need to just deprecate it.

Reply:
FWIW, I'm all for deleting go.sum support now. I don't see a strong case for deprecation of go.sum support.